gms/api/src/main/pegasus/com/linkedin/dataprocess/DataProcess.pdl

```diff
@@ -0,0 +1,17 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.DataProcessUrn
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.Uri
+
+/**
+ * Spec. for a process. A collection of data conforming to a single schema that can evolve over time.
+ */
+record DataProcess includes DataProcessKey {
+
+  /**
+  * The data process info includes the datasets of input & output
```

Comment:
Can rephrase it to "Metadata associated with a data process including input and output datasets"